R/track-util.R

```diff
@@ -0,0 +1,142 @@
+#' track helpers
```

Comment:
I removed the track-utils and put all functions in track.R. I suggest to restructure the code with rising  complexity .